js/dialogcards.js

```diff
@@ -70,26 +92,138 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
       '<div class="h5p-dialogcards-description">' + self.params.description + '</div>'
       ));
 
+      // If we are resuming task from a previously finished task, ask user it they want to Retry.
+      if (this.taskFinished) {
+        self.finishedScreen();
+        return;
+      }
+      var existsCardOrder = true;
+      if ($.isEmptyObject(this.cardOrder)) {
+        existsCardOrder = false;
+      }
+      // Create cardOrder and cardNumber buttons only on first instanciation for logged in user.
+      if (self.params.behaviour.randomCards == 'user' && !existsCardOrder) {
+        self.createOrder().appendTo(self.$inner);
+      } else {
+        self.attachContinue();
+      }
+  };
+
+    /**
+   * Attach the rest of the h5p inside the given container.
+   *
+   * @param {jQuery} $container
+   */
+   C.prototype.attachContinue = function ($container) {
+    var self = this;
+
+    // Remove potential user interaction elements from DOM.
+    var $el = '.h5p-dialogcards-number';
+    $( $el ).remove();
```

Comment:
I'm sure it would be possible to set up scenarios where this does cause problems. Quickest way to make it work would be something like $('.h5p-dialogcards-number', self.$inner) since a dialog card probably never will contain another dialog card as a child. If you provide a context that is closer to the element you're searching for it will be better (faster) and even better if you can skip jQuery and use a reference instead.

(Btw. fyi the core team is moving away from jQuery and doing native js instead)